File: WCF/Windows SDK WCF Samples/TechnologySamples Beta 1/Scenario/LoadBalancer/VB/service1/service1.vb
' Copyright (c) Microsoft Corporation.  All Rights Reserved.

Imports System
Imports System.Xml
Imports System.ServiceModel
Imports System.Configuration

Namespace Microsoft.ServiceModel.Samples

    <ServiceContract([Namespace]:="http://Microsoft.ServiceModel.Samples")> _
    Public Interface ILoadBalancedEchoContract

        <OperationContract()> _
        Function Echo(ByVal message As String) As String

    End Interface

    Public Class LoadBalancedEchoService
        Implements ILoadBalancedEchoContract

        Public Function Echo(ByVal message As String) As String Implements ILoadBalancedEchoContract.Echo

            Return message

        End Function

    End Class

End Namespace

File: WCF/Windows SDK WCF Samples/TechnologySamples Beta 1/Scenario/LoadBalancer/CS/service1/service1.cs

//  Copyright (c) Microsoft Corporation.  All Rights Reserved.

using System;
using System.Xml;
using System.ServiceModel;
using System.Configuration;

namespace Microsoft.ServiceModel.Samples
{
    [ServiceContract(Namespace = "http://Microsoft.ServiceModel.Samples")]
    public interface ILoadBalancedEchoContract
    {
        [OperationContract]
        string Echo(string message);
    }

    public class LoadBalancedEchoService : ILoadBalancedEchoContract
    {
        public string Echo(string message)
        {
            return message;
        }
    }
}
